tests/risk/test_risk_period.py

```diff
@@ -385,7 +385,7 @@ def test_benchmarkrange(self):
     def test_partial_month(self):
 
         start_session = self.trading_calendar.minute_to_session_label(
-            pd.Timestamp("1991-01-01", tz='UTC')
+            pd.Timestamp("1993-02-01", tz='UTC')
```

Comment:
I made this change because the furthest date for which we have data from Google Finance is `1993-02-01`, so this test fails because it's looking for data at a start date before the oldest date.